apps/omg_watcher/priv/repo/migrations/20190925132500_add_and_fix_timestamps.exs

```diff
@@ -0,0 +1,36 @@
+defmodule OMG.Watcher.Repo.Migrations.AddAndFixTimestamps do
+  use Ecto.Migration
+
+  def change do
```

Comment:
Without myself having tested this, I want to note that this will probably fail on rollbacks as it would not have enough info to reconstruct `:inserted_at`, `updated_at` fields, etc. If rollback needs to be supported `change/0` here will need to be broken down to `up/0` and `down/0`.

Reply:
I don't think rollback needs to be supported. But even if we wanted to, when a column is deleted, there's no way to recover the data and thus the `inserted_at` and `updated_at` columns cannot be reconstructed (at least not with the previous data the columns contained).

I'm fully taking advantage of the fact that we can get away with this until mainnet. 🙂